tests/testthat/test-plot_with_settings_ui.R

```diff
@@ -6,39 +6,118 @@ testthat::test_that("plot_with_settings_ui returns `shiny.tag.list`", {
   testthat::expect_s3_class(plot_with_settings_ui("STH"), "shiny.tag.list")
 })
 
-testthat::test_that("Plot with settings: UI screenshots", {
-  app <- shinytest2::AppDriver$new(
-    app_pws(),
-    name = "pws",
-    variant = "app_pws_ui"
-  )
-  threshold <- 75
-  kernel_size <- 5
-  delay <- 0.1
-
-  app$set_window_size(width = 1000, height = 700)
-
-  # click on hide/show button
-  app$click("button")
-  app$expect_screenshot(
-    threshold = threshold, kernel_size = kernel_size, delay = delay, name = "hidden"
-  )
-  app$click("button")
-  app$expect_screenshot(
-    threshold = threshold, kernel_size = kernel_size, delay = delay, name = "visible"
-  )
-
-  app$set_inputs(`plot_with_settings-downbutton-file_name` = "plot1")
-  app$set_inputs(`plot_with_settings-modal_downbutton-file_name` = "plot2")
-
-  app$click("plot_with_settings-downbutton-downl")
-  app$expect_screenshot(
-    threshold = threshold, kernel_size = kernel_size, delay = delay, name = "download_menu"
-  )
-  app$click("plot_with_settings-expbut")
-  app$expect_screenshot(
-    threshold = threshold, kernel_size = kernel_size, delay = delay, name = "resize_menu"
-  )
-
-  app$stop()
-})
+testthat::test_that(
+  "e2e: teal.widgets::plot_with_settings is initialized with proper plot and buttons",
```

Comment:
We don't really know if the plot is proper. Word proper is exaggerated. I would say
```suggestion
  "e2e: teal.widgets::plot_with_settings is initialized with a plot and toggle, download and resize buttons",
```

And I would limit test to this only. Then I would test:

- clicking download button opens popup containing this and that
- clicking expand button opens modal containing this and that
- clicking resize button opens popup containing this and that

Then you can do to details
- clicking download+download button downloads image in a specified format
- expanded image can be resized
- expanded image can be downloaded
- image can be resized